src/test/java/org/example/pageobjectstests/DeliveryToFunctionality.java

```diff
@@ -0,0 +1,28 @@
+package org.example.pageobjectstests;
+
+import org.example.factory.WebDriverFactory;
+import org.example.pageobject.deliverytofunctionalitypages.HomePage;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+public class DeliveryToFunctionality {
+
+    public HomePage homePage;
+    public String poland = "Poland";
+    public String zip = "90210";
+
+    @BeforeClass
+    public void setup() {
+        WebDriver webDriver = new WebDriverFactory().getWebDriver();
+        homePage = new HomePage(webDriver);
+    }
+
+    @Test
+    public void testDeliverCountryPoland() {
+        boolean  isDeliveredCountryPoland =  homePage.open().openProfileDropDown().inputPostalCodeInDropdownModuleAndClickApplyButton("90210").openProfileDropDown().
```

Comment:
Why do you use a hardcore "90210", if you have a variable with the same value? The same for "Poland"